azure-iot-device/azure/iot/device/abomination.py

```diff
@@ -0,0 +1,227 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module provides hard coded patches used to modify items from the libraries.
+Currently we have to do like this so that we don't use exec anywhere"""
+
+
+def execute_atrocity_for_sync():
```

Comment:
>execute_atrocity_for_sync [](start = 4, length = 25)

I actually Laughed Out Loud

Reply:
:D

---
In reply to: [389162134](https://github.com/Azure/azure-iot-sdk-python/pull/482#discussion_r389162134) [](ancestors = 389162134)

Reply:
I also laughed, but...

<buzzkill>
this naming probably isn't a great look for us from an open source perspective. 
Just having these files alone is enough of an atrocity on the codebase.
</buzzkill>

---
In reply to: [389197704](https://github.com/Azure/azure-iot-sdk-python/pull/482#discussion_r389197704) [](ancestors = 389197704,389162134)